app/src/main/res/values-de/strings.xml

```diff
@@ -6,7 +6,7 @@
     <string name="action_settings">Einstellungen</string>
     <string name="action_logout">Abmelden</string>
     <string name="action_search">Suchen</string>
-    <string name="action_updates">Aktualisierungen</string>
+    <string name="action_updates">Updates</string>
```

Comment:
In most linux distros they use "Aktualisierungen", so it felt more consistent to me. And yea it wasn't consistent within this translation (what it shoult be). :)